cms/settings/base.py

```diff
@@ -81,7 +81,7 @@
     "cms.taxonomy",
     "cms.search",
     "cms.workflows",
-    "wagtail.embeds",
+    "wagtail.embeds",  # TODO: Now that the ONSEmbedBlock is gone, I am guessing I can remove this?
```

Comment:
Even `ONSEmbedBlock` never used it. It's used when using: https://docs.wagtail.org/en/7.0/advanced_topics/embeds.html